src/Reflection/ReflectionClass.php

```diff
@@ -540,17 +540,17 @@ public function getConstant(string $name): string|int|float|bool|array|null
      */
     public function hasConstant(string $name): bool
     {
-        return $this->getReflectionConstant($name) !== null;
+        return $this->getConstant($name) !== null;
     }
 
     /**
      * Get the reflection object of the specified class constant.
      *
      * Returns null if not specified.
      */
-    public function getReflectionConstant(string $name): ?ReflectionClassConstant
+    public function getConstant(string $name): ?ReflectionClassConstant
```

Comment:
In order to ease the migration to `5.0`, I think we should have a different name for any symbols that changed return type, or else the BC break becomes more subtle.

Yes, that muds things a bit, since we'd have `get*ReflectionConstant` or such, but it's still better than a hard break at type level, with the same name :thinking:

Reply:
I'm not sure if the "cleanup" make sense then. Now it could make `getProperty()`, `getMethod()` and `getConstant()` consistent...

Reply:
Let's leave it be for now then?

Reply:
Ok :)